Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ChannelGroup/ByChannelGroup.tsx

```diff
@@ -84,86 +62,59 @@ const ChannelGroups: Application.Types.iByComponent = (props) => {
         setErrors(e);
     }, [record]);
 
+    function addNewChannelGroup() {
+        let handle = $.ajax({
+            type: "POST",
+            url: `${homePath}api/ChannelGroup/Add`,
+            contentType: "application/json; charset=utf-8",
+            data: JSON.stringify(record),
+            dataType: "json",
+            cache: false,
+            async: true
+        }).done(() => {
+            refreshData(x => x + 1);
+        })
+
+        return () => {
+            if (handle != null && handle.abort != null) handle.abort();
+        };
```

Comment:
Should be a Controller call.
Unless you call this in the `useEffect` cleanup won't do anything and should be removed